include/ck_tile/ops/fmha/kernel/fmha_fwd_splitkv_kernel.hpp

```diff
@@ -616,10 +597,11 @@ struct FmhaFwdSplitKVKernel
                     sequence<kPadSeqLenQ, kPadHeadDimQ>{});
             }
         }();
-        const auto k_dram = [&]() {
+
+        const auto make_k_dram = [&](const KDataType* data, index_t height) {
```

Comment:
Seems this pr doesn't support group mode. So if i want to customize this for group mode, we could add a Q dram navigator to implement it?

Reply:
I think the only difference between batch & group mode is final `kargs.seqlen_q` value & `batch_offset_q` computation logics. In order to support group mode, you will have to set `kargs.seqstart_q_ptr` array and check its value in `*_kernel.hpp`

Reply:
OK, thanks. I will have a try.